src/components/pages/TeamSchedule.vue

```diff
@@ -172,14 +182,19 @@ export default {
       'assignSelectedTasks',
       'fetchPersonTasks',
       'getPersonsTasksDates',
+      'loadDaysOff',
       'loadPeople',
       'unassignPersonFromTask',
       'updateTask'
     ]),
 
     async init() {
       this.loading.schedule = true
-      await Promise.all([this.loadPeople(), this.loadPersonDates()])
+      await Promise.all([
```

Comment:
Running requests parallelly often leads to unexpected behavior. Can you run them sequentially?

Reply:
done!